"""
Задание: пропуск тестов
Изучите самостоятельно документацию про маркировку xfail. Найдите там параметр, который в случае неожиданного
прохождения теста, помеченного как xfail, отметит в отчете этот тест как упавший. Пометьте таким образом первый тест
из этого тестового набора.

test_xfail.py:
"""

import pytest

@pytest.mark.xfail(strict=True)
def test_succeed():
    print('\nНачало Succeed теста')
    assert True

@pytest.mark.xfail
def test_not_succeed():
    print('\nНачало Not succeed теста')
    assert False

@pytest.mark.skip
def test_skipped():
    print('\nНачало Skipped теста')
    assert False

"""
Запустите полученные тесты. Обратите внимание на статус прогона тестов. Найдите последнюю строчку с итогами запуска, 
скопируйте текст между символами === и отправьте его в качестве ответа на это задание. 
"""

"""
Параметр strict
Ни XFAIL, ни XPASS по умолчанию не приводят к падению всего набора тестов. Но это можно изменить, установив параметру 
strict значение True:

@pytest.mark.xfail(strict=True)
def test_function():
    ...
В этом случае, если тест будет неожиданно пройден (XPASS), то это приведет к падению всего тестового набора.

Значение по умолчанию параметра strict можно изменить в настройках (в файле``pytest.ini``), 
используя опцию xfail_strict:

[pytest]
xfail_strict=true
"""